### ISSUE TITLE
Add more deployment options for Netlify

### ISSUE THREAD

**Issue by jharrell** [prisma/prisma2-docs#5141]
After #5088 the only method described is linking a GitHub repo. We should discuss other options, specifically the [deploy local files option](https://docs.netlify.com/welcome/add-new-site/#deploy-local-files).

These updates are less important than updating all guides generally, but more important than creating new guides that would be only used in a small capacity.

**Comment by nikolasburk** [comment]
Closing this as not planned, we can re-open another issue like this if this becomes relevant again.
